config/config.go

```diff
@@ -39,21 +40,39 @@ const (
 )
 
 const (
-	_appRoot     = "APP_ROOT"
-	_environment = "_ENVIRONMENT"
-	_datacenter  = "_DATACENTER"
-	_configDir   = "_CONFIG_DIR"
-	_configRoot  = "./config"
-	_baseFile    = "base"
-	_secretsFile = "secrets"
+	_appRoot            = "APP_ROOT"
+	_environment        = "_ENVIRONMENT"
+	_datacenter         = "_DATACENTER"
+	_mesosContainerName = "MESOS_CONTAINER_NAME"
+	_configDir          = "_CONFIG_DIR"
+
+	_configRoot       = "./config"
+	_baseFile         = "base"
+	_deploymentPrefix = "deployment"
+	_monopod          = "monopod"
+	_secretsFile      = "secrets"
 )
 
 var (
 	_setupMux sync.Mutex
 
 	_envPrefix            = "APP"
+	_deploymentEnv        = "APP_DEPLOYMENT_NAME"
```

Comment:
Its not _env_prefix in the backend, but completely different environment variable. I'll update.